ipld/cid/src/lib.rs

```diff
@@ -35,28 +29,63 @@ const MULTIBASE_IDENTITY: u8 = 0;
 #[cfg(feature = "json")]
 pub mod json;
 
-/// Representation of a IPLD CID.
-#[derive(PartialEq, Eq, Clone, Copy)]
-pub struct Cid {
-    pub version: Version,
-    pub codec: Codec,
-    pub hash: Multihash,
+pub const DAG_CBOR: u64 = 0x71;
+pub const DAG_PB: u64 = 0x70;
```

Comment:
This doesn't have to be an exposed constant, it's not used for anything meaningful (just a test that could use any value)

Reply:
Changed tests to `DAG_CBOR`.